src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLRenderChoiceParamsStrategy.java

```diff
@@ -0,0 +1,48 @@
+package com.adq.jenkins.xmljobtodsl.dsl.strategies.custom;
+
+import com.adq.jenkins.xmljobtodsl.dsl.strategies.DSLParameterStrategy;
+import com.adq.jenkins.xmljobtodsl.parsers.PropertyDescriptor;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DSLRenderChoiceParamsStrategy extends DSLParameterStrategy {
+
+    private final String name;
+
+    public DSLRenderChoiceParamsStrategy(PropertyDescriptor propertyDescriptor) {
+        super(propertyDescriptor);
+        this.name = "";
+    }
+
+    public DSLRenderChoiceParamsStrategy(int tabs, PropertyDescriptor propertyDescriptor, String name) {
+        super(propertyDescriptor);
+        this.name = name;
+    }
```

Comment:
come to think of it we never use the name elsewhere, we can probably just totally ignore it. Java just wants us to be able to take it as a parameter that's all

Reply:
Gotcha, I left that as a parameter and removed the rest.